### ISSUE TITLE
[account-web-vault] Login when already on phone doesn't work

### ISSUE THREAD

**Issue by tahpot** [verida/verida-js#46]
The `Login with Verida Vault` link does nothing when clicked.

This needs to redirect to the same URL that is embedded in the QR code.

![Screen Shot 2021-09-18 at 2 14 55 pm](https://user-images.githubusercontent.com/164973/133872821-1d52d985-62e0-42c0-8d54-d543a6d31bbf.png)
nk

Also, is there any way we can detect you're on a mobile device and auto redirect if you are?

**Comment by cmcWebCode40** [comment]
Yes there's a way we can detect if the user is on a mobile screen, either using `window.media()` or the material UI framework also has a function that does that too

**Comment by nick-verida** [comment]
See also https://github.com/verida/vault-auth-landing-page/issues/8

**Comment by saadibrahim** [comment]
Update: We need to show "Open Vault" button instead of QR code on Mobile.

**Comment by saadibrahim** [comment]
Notes from meeting on this:

- conclusion: two-click solution because of constraints, its also faster
- the vault URL is created by creating a socket connection when the connect button is pressed
- we cannot make a socket connection on page load as that will create a connection on each page load
- universal links/deep links dont work with redirects, have to click to open app otherwise it just opens in the browser
- we cant play a lot with the connect page as its not part of the verida library, what happens after pressing connect button is part of the verida library
- we could send the credentials to the app and do the socket connection there but that would put more burden on the app
